packages/shell-api/src/shell-api.ts

```diff
@@ -0,0 +1,133 @@
+import AsyncWriter from '@mongosh/async-rewriter';
+import Mapper from '@mongosh/mapper';
+import { types, Help } from '@mongosh/shell-api-types';
+
+class ShellApi {
+  public mapper: any;
+  private asyncWriter: AsyncWriter;
+  private bus: any;
+  private container: any;
+  constructor(serviceProvider, bus, container) {
+    this.mapper = new Mapper(serviceProvider, bus);
+    this.asyncWriter = new AsyncWriter({}, types);
+    this.bus = bus;
+    this.container = container;
```

Comment:
So the container is only used for telemetry right now, eh? I wonder if we could default it to noop, so you don't have to do [those checks below](https://github.com/mongodb-js/mongosh/pull/103/files#diff-3c80e6a0146b17d9c12433c0f55c1147R62)?

Reply:
Yeah, I thought there might be chance there will be other things we need to do beyond telemetry. Can you think of any?

Reply:
I missed this! Sorry! 

Nothing comes to mind right now aside from telemetry.